packages/angular-material/src/controls/autocomplete.renderer.ts

```diff
@@ -96,6 +97,7 @@ import { startWith } from 'rxjs/operators/startWith';
 export class AutocompleteControlRenderer extends JsonFormsControl {
   @Input() options: string[];
   filteredOptions: Observable<string[]>;
+  shouldNotFilter: boolean;
```

Comment:
Changed :grin: 
I also change the event type that is used for updating the filter from `key up` to `key down`, as the first key was otherwise ignored.